src/app/main-page/presentation/toolbar-action/toolbar-action.component.html

```diff
@@ -6,8 +6,22 @@
         *ngIf="fileName !== null"
         class="fcl-file-name-wrapper fcl-separator mat-body"
     >
-        <div class="fcl-file-name" [matTooltip]="fileName">
-            File: {{ fileNameWoExt }}
+        <div class="fcl-file-name-inner-wrapper">
```

Comment:
hmm I think it is necessary: `.fcl-file-name-wrapper` needs to be larger so that all the elements in the toolbar are correctly aligned. the warning sign needs to be positioned based on the actual size of `fcl-file-name`. `fcl-file-name` needs `overflow-x: hidden` so that `elipsis()` works. Unfortunately, we cannot combine `overflow-x: hidden` and `overflow-y: visible`, because visible gets computed to auto if combined with hidden (so it's basically `overflow: hidden`) and then the warning sign would be cut off ...